aix.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const getSqlStatement = family => {
+  let sql =
+    "select NEXT_HOP from QSYS2.NETSTAT_ROUTE_INFO where ROUTE_TYPE='DFTROUTE'";
```

Comment:
Does this table possibly also contain the name of the network interface? If so, we can fill the `interface` property of the result with it.

Reply:
According to [this](https://www.ibm.com/developerworks/community/wikis/home?lang=es-es#!/wiki/IBM%20i%20Technology%20Updates/page/QSYS2.NETSTAT_ROUTE_INFO) it might be `LOCAL_BINDING_INTERFACE` but it says it will be null for IPv6, so not sure if worth to add.

Reply:
I‘m gonna have a look on it tomorrow. If it‘s null, I‘m gonna make sure it‘s at least an empty string, using coalesce, or maybe there‘s some other way round.

Reply:
Yeah result should be non-empty string or null. Guess it does have some value to have it at least in the IPv4 case.

Reply:
I checked the `LOCAL_BINDING_INTERFACE`, and it contains, indeed, the interface address. So I added it.